src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -820,12 +818,22 @@ function getVisibilityMessage(buttonElem) {
         url: url,
         data: formData,
         success: function(data) {
+            // update stored form data
+            previousFormDataMap[qnNumber] = formData;
+            
             $formVisibility.html(formatVisibilityMessageHtml(data.visibilityMessage));
             $formVisibility.show();
             $formOptions.hide();
         },
         error: function(jqXHR, textStatus, errorThrown) {
-
+            var visibilityButton = $form.find('.visibilityMessageButton');
+            var radioInput = visibilityButton.children().first();
```

Comment:
I am guessing this code is needed, but I don't know why? Leave a comment in the code, if needed. 

Using children.first might not be the best way to get a radio input, might be better if you just explicitly select the radio input, if the radio input is what you need here.